protoc-gen-gql/main.go

```diff
@@ -255,7 +263,12 @@ func (p *plugin) renderMethod(methods []*Method) {
 			}
 		} else if typ, ok := p.Types()[m.OutputType]; ok {
 			if !p.IsEmpty(typ) {
-				out = p.GqlModelNames()[typ]
+				if p.unwrapResponse {
+					field := typ.GetField()[0]
```

Comment:
this unwraps the response but the old response types are still generated even though they are not used anywhere